src/cloud/components/Comments/CommentList.tsx

```diff
@@ -44,7 +46,12 @@ function CommentList({
   }, [comments])
 
   return (
-    <div className={className}>
+    <CommentListContainer className={className}>
+      {initialComment == null && (
+        <div className={'deleted_initial_comment'}>
```

Comment:
lets not abuse brackets for strings
```
<div className='deleted_initial_comment'>
```

className feels a bit overkill to me

`comment--deleted` would be simpler and maybe more reusable.